src_assets/windows/assets/apps.json

```diff
@@ -1,17 +1,15 @@
 {
-  "env": {
-    "PATH": "$(PATH);$(ProgramFiles(x86))\\Steam"
-  },
+  "env": {},
```

Comment:
I'll review this in more detail tomorrow... but shouldn't we leave `PATH": "$(PATH)` only removing the default Steam path?

Reply:
`"PATH": "$(PATH)` should be equivalent to having no `env` block. We take all the environment variables from the user then apply any additions or modifications from the `env` block, so if the `env` block is empty, the app just gets the user's unmodified environment variables.

Reply:
Perfect!